tool/grind.dart

```diff
@@ -208,7 +208,8 @@ updateTestPackageDocs() {
         '../test_package_docs',
         '--example-path-prefix',
         'examples',
-        '--auto-include-dependencies'
+        '--auto-include-dependencies',
+        '--pretty-index-json'
```

Comment:
This diff is *huge* so it's collapsed - https://github.com/dart-lang/dartdoc/pull/1355/files#diff-613b594c67e5864627539fcd158f63fc